src/components/about/FourPillTitle.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const Heading = () => {
+  return (
+    <div className="flex justify-end">
+      <div className="flex-col">
+        <p className="w-fit pr-8 text-3xl font-semibold text-csa-red-100">
```

Comment:
```suggestion
        <p className="w-fit pr-8 text-3xl font-semibold text-csa-red-100 border-b">
```

and then add the styling needed. also make the text for four pillars a bit larger, other than that your code's good to go